### PR TITLE
Feature nitika

### PR DESCRIPTION
**Pull request by nitika807**
I have added some text to the file.

### CODE REVIEW

**Comment by Surinder07** [comment]
Please add couple of more reviewers to this repo.

**Review by Surinder07 — APPROVED** [review]
Please add two more reviewers here

**Comment by nitika807** [comment]
> Please add two more reviewers here

I have asked others. but they seem busy and are not replying. I'll try to ask others as well.

**Comment by Surinder07** [comment]
> > Please add two more reviewers here
> 
> I have asked others. but they seem busy and are not replying. I'll try to ask others as well.

No worries, will see in tomorrows class

File: index.html [hunk 1]
```diff
@@ -5,6 +5,6 @@
     <title>Title</title>
 </head>
 <body>
-
+This is the home page of the website.
```
Comment (nitibilkhu) [inline index.html:8]: Make **HEADER** and add nav bar and search in the header.

File: main.css [hunk 2]
```diff
@@ -5,6 +5,6 @@
     <title>Title</title>
 </head>
 <body>
-
+This is an example to execute the assignment 2 on Git repositories.
```
Comment (nitibilkhu) [inline main.css]: Can you please add  more lines in this file

File: index.js [hunk 3]
```diff
@@ -5,6 +5,6 @@
     <title>Title</title>
 </head>
 <body>
-
+//This document consists of the javascript codes for the webpage
```
Comment (nitibilkhu) [inline index.js:14]: Add dynamic content in this file

**Review by nitibilkhu — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (nitika807) [reply index.html:8]: I have added. Please review it.

File: main.css (hunk 2 above)
Comment (nitika807) [reply main.css]: lines have been added.

File: index.js (hunk 3 above)
Comment (nitika807) [reply index.js:14]: I will add dynamic content in the coming stages of the project. For now, I have not added anything in this file.

File: main.css [hunk 4]
```diff
@@ -3,8 +3,25 @@
...
+     }
+
+     p {
+     background-color: yellow;
+     }
+     </style>
 </head>
 <body>
-
+<h1>Practicing Git Commands</h1>
```
Comment (nitibilkhu) [inline main.css:21]: Heading tag, paragraph tag and div tag is added with styling and content.

**Review by nitibilkhu — CHANGES_REQUESTED** [review]

**Review by nitibilkhu — APPROVED** [review]

**Comment by nitika807** [comment]
The reviews has been performed by the team members and a merge is going to be performed into main branch.

**Pull request merged**